examples/kubernetes/lib/custom_filter.py

```diff
@@ -0,0 +1,4 @@
+import base64
+
+def custom_filter(string):
+    return base64.b64encode(string.encode("UTF-8")).decode("UTF-8")
```

Comment:
Adding 1-2 comment lines explaining what this example filter does, would be a good practice.